commentHandler.go

```diff
@@ -256,21 +263,78 @@ func manageLocking(req types.IssueCommentOuter, cmdType string) (string, error)
 	return buffer.String(), nil
 }
 
+func manageMilestone(req types.IssueCommentOuter, cmdType string, cmdValue string) (string, error) {
+
```

Comment:
remove some empty lines across the function. a good rule of thumb is to usually combine blocks of related code.
so if some lines are part of the same logic, combine them without empty lines. if these are about separate logic add empty lines between the separate logic.

for instance:
```
fun someFunc() {
	// here we only declare vars without empty lines between them
	var param1 string
	var param2 string
	someValue := something

	// some logic here. add an empty line on top to separate it
	if (someValue == "test") {
		// ...
	}
	param2 = "test2"

	// separate logic, so empty line separating it, again.
	if (param1 == "test2") {
		// ...
	}
}
```